### DISCUSSION TITLE
Documentation for `WCSGetCoverage` does not match arguments available through `wcs$getCoverage`

### DISCUSSION THREAD

**Discussion by annakrystalli** [eblondel/ows4R#89] [OGC WCS] [answered]
I'm having difficulty understanding the arguments to `wcs$getCoverage()` as they appear to differ from those shown in the help file for `WCSGetCoverage`.

Here's a screenshot of the rstudio popup box with function arguments for `wcs$getCoverage`
<img width="657" alt="image" src="https://user-images.githubusercontent.com/5583057/186373194-c724e5b8-8e0c-4ab0-99d0-e74d9f5a519e.png">

whereas here is the argument list in the `WCSGetCoverage` help file:

<img width="534" alt="image" src="https://user-images.githubusercontent.com/5583057/186373985-2b9349c8-1ed2-4953-83e4-00faf364565d.png">

Additionally, the argument descriptions are extremely sparse, basically just a reprinting of the argument name which doesn't help a user understand what the arguments are, what type of input they expect (numeric? character? vector? bbox?) and how/for what they should be used.

**Comment by eblondel** [comment, marked as answer]
`WCSGetCoverage` is a class not meant to be used directly by user, but is used internally by ows4R WCS client base on its capabilities. It shares coverage request parameters with the one exposed in `WCSCoverageSummary`, but not all - by design- to make it usable from clients and to allow inheriting information that are given by WCS capabilities.

`getCoverage` in `WCSClient` has been aligned with `getCoverage` in `WCSCoverageSummary` . Some parameters were described. Extra parameters will be further documented in time
